pkg/handler/dashboard/dashboard.go

```diff
@@ -130,3 +130,77 @@ func (h *handler) WorkSurveys(c *gin.Context) {
 
 	c.JSON(http.StatusOK, view.CreateResponse[any](view.ToWorkSurveyData(project, workSurveys), nil, nil, nil, ""))
 }
+
+// GetActionItemReports godoc
+// @Summary Get Action items report for dashboard
+// @Description Get Action items report for dashboard
+// @Tags Dashboard
+// @Accept json
+// @Produce json
+// @Param Authorization header string true "jwt token"
+// @Param projectID   query  string false  "Project ID"
+// @Success 200 {object} view.ActionItemReportData
```

Comment:
it should create new object ActionItemReportResponse
```
"data": {
        "auditActionItemReports": []
}
```
because the response is the value of  "data" key

Reply:
Updated, thanks